lib/inspec/utils/parser.rb

```diff
@@ -109,8 +121,16 @@ def parse_mount_options(mount_line, compatibility = false)
 
       # Device-/Sharename or Mountpoint includes whitespaces?
       def includes_whitespaces?(mount_line)
-        ws = mount_line.match(/^(.+)\son\s(.+)\stype\s.*$/)
-        ws.captures[0].include?(" ") || ws.captures[1].include?(" ")
+        # Split the mount_line by " on "
+        parts = mount_line.split(" on ")
+
+        # Check if the first part (device/share) or the second part (path) contains " type "
```

Comment:
where is this comment being satisfied?

Reply:
please ignore this. Forgot to remove this during the cleanup.